modules/business-text/business-text.graphql.js

```diff
@@ -0,0 +1,20 @@
+const businessTextType = `
+    type BusinessText {
+    _id: ID!
+    code: String
+    title: [Language]
+    text: [Language]
+    languages: [String]
+    date: String
+}
+`;
+
+const businessTextInput = `
+    input BusinessTextInput {
+    code: String
```

Comment:
I don't think that it should be nullable

Reply:
done